firedrake/ufl_expr.py

```diff
@@ -115,7 +115,7 @@ def TrialFunctions(function_space):
     return split(TrialFunction(function_space))
 
 
-def derivative(form, u, du=None):
+def derivative(form, u, du=None,coefficient_derivatives=None):
```

Comment:
To elaborate, our continuous integration system checks for certain style violations in the codebase.  You can check this locally by running `make lint` in the firedrake source directory, assuming the `flake8` tool is installed.  This failure will be because we require a space after the comma.